memtier_benchmark.cpp

```diff
@@ -400,6 +401,7 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
         { "port",                       1, 0, 'p' },
         { "unix-socket",                1, 0, 'S' },
         { "protocol",                   1, 0, 'P' },
+        { "resp",                       1, 0, o_resp },
```

Comment:
Did you consider extending `--protocol` instead of adding another flag?

Reply:
Yes, but for me, it seems more internal configuration under the Redis protocol. and for backport compatibility, we probably need to add another 2 options (`redis-resp2`, `redis-resp3`) as the existing one refers to not set `resp` version at all.